openmdao/solvers/nonlinear/nonlinear_runonce.py

```diff
@@ -33,17 +30,21 @@ def solve(self):
         """
         system = self._system
 
-        # If this is a parallel group, transfer all at once then run each subsystem.
-        if len(system._subsystems_myproc) != len(system._subsystems_allprocs):
-            system._transfer('nonlinear', 'fwd')
-            for subsys in system._subsystems_myproc:
-                subsys._solve_nonlinear()
-            system._check_reconf_update()
-        # If this is not a parallel group, transfer for each subsystem just prior to running it.
-        else:
-            for isub, subsys in enumerate(system._subsystems_myproc):
-                system._transfer('nonlinear', 'fwd', isub)
-                subsys._solve_nonlinear()
+        from openmdao.recorders.recording_iteration_stack import Recording
```

Comment:
I recall it being there because it caused a circular, but I will try it again in case that is old info.  Also, any suggestions for re-arranging the imports are welcome!